Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -389,7 +389,26 @@ def bottle_formula(f, args:)
           sudo_purge
           # Unset the owner/group for reproducible bottles.
           # Tar then gzip for reproducible bottles.
-          safe_system "tar", "--create", "--numeric-owner", "--file", tar_path, "#{f.name}/#{f.pkg_version}"
+          owner_group_args = if OS.mac?
+            if MacOS.version < :catalina
+              ["--owner", "0", "--group", "0"]
+            else
```

Comment:
I don't quite follow this. You're installing `gnu-tar` on macOS versions newer than Mojave. Didn't you want the reverse?

Reply:
@carlocab YUP 🤦🏻